style.css

```diff
@@ -281,3 +281,75 @@ button {
   height: 7%;
   margin-left: auto;
 }
+
+/* Contact me form  */
+.contact-me-form {
+  background-color: #6070ff;
+  background-image: url("images/contact-shape-mobile.svg");
+  z-index: -1;
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: top right;
+  margin-top: 100px;
+  border: none;
+  border-top-left-radius: 100px;
+}
+
+.contact-form {
+  padding: 40px 30px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.contact-form h4 {
+  color: #fff;
+  font-size: 40px;
+  margin-bottom: 0;
+}
+
+.contact-form p {
+  color: #ebebff;
+  font-size: 20px;
+  text-align: center;
+}
+
+.contact-form form {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  padding-top: 20px;
+  width: 100%;
+}
+
+.contact-form form > input {
+  height: 40px;
+  border: none;
+  border-radius: 5px;
+}
+
+.contact-form form > input::placeholder {
+  color: #172b4d;
+  font-size: 17px;
+  font-family: 'Poppins', sans-serif;
```

Comment:
- [ ] [OPTIONAL] For increasing 🆙 your chances in the job search 💼 after finishing microverse, we request that you follow coding best practices 💯  in all of your projects. One of these practices is **DRY** (**D**on't **R**epeact **Y**ourself). That means if you find yourself repeating 🔁 a specific line of code in multiple places, you have to find a way to only use it once 1️⃣. This way when you update ♻️  this line, you won't have to remember where else did you write that to update the other copies.

### You can check this in the [Html & Css Best Practices Page](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

![image](https://user-images.githubusercontent.com/60979926/164267978-27c9c5e1-37bb-4d2d-81ef-e7cd491bfbdb.png)

### In our case

you can easily use a `.font` class in multiple places or simply put the `font-family` property in the `body` selector and all other children elements going to inherit it by default.

- (your code)

```css
.contact-form form textarea::placeholder {
  /* ...code */
  font-family: inter, sans-serif;
  /* ...code */
}

.contact-form form > input::placeholder {
  /* ...code */
  font-family: inter, sans-serif;
  /* ...code */
}
```

- suggestion

```css
.font {
  font-family: inter, sans-serif;
}
```

> **Note**: Also, you can do the same thing with the `display: flex` declaration.

Reply:
Thanks. This is actually genius. i'll apply the same class instead!

Reply:
@MrRamoun that font is for the placeholder. When i add it as a class to the input and textarea tags, there is actually no effect. What suggestion do you have?

Reply:
@AbbyNyakara you can use id instead of class. I believe it is a specificity issue